tests/integration/test_batch.py

```diff
@@ -80,3 +89,29 @@ def test_export_data_rows(configured_project: Project, dataset: Dataset):
 
     assert len(result) == n_data_rows
     assert set(data_rows) == set(exported_data_rows)
+
+
+@pytest.mark.skip(
+    reason="Test cannot be used effectively with MAL/LabelImport. \
```

Comment:
Can you investigate the reason behind this skip, and make sure that we have this fix lined up as a jira ticket?

Reply:
This is likely interaction between `Batch` vs what we expect from `Dataset` since they work so differently. We're seeing a similar issue when trying to even create labels in a batch project vs a dataset project. I'll create a follow up ticket for this as a future SDK incidental

Reply:
Created polish ticket here. will be pushing forward with these changes
https://labelbox.atlassian.net/browse/AL-3261